config/environment.js

```diff
@@ -100,7 +100,7 @@ module.exports = function (environment) {
 			// when it is created
 		},
 		fastboot: {
-			hostWhitelist: [/.*\.wikia-dev\.(pl|us)/, /.*\.wikia-staging.com/, /.*\.wikia.com/],
+			hostWhitelist: [/.*\.wikia-dev\.(pl|us)/, /.*\.wikia-staging.com/, /.*\.wikia.com/,  /^localhost:\d+$/],
```

Comment:
it's related to an error occurring while trying to run docker image locally, not sure if we want it now (or ever)